Setting.css

```diff
@@ -0,0 +1,123 @@
+:root {
+    --font-family: 'Poppins', sans-serif;
+
+    --color-white: #f3f3f3;
+    --color-darkblue: #1b1b32;
+    --color-darkblue-alpha: rgba(27, 27, 50, 0.8);
+    --color-green: #37af65;
+
+    --accent-left: rgba(58, 58, 158, 0.8);
+    --accent-right: rgba(136, 136, 206, 0.7);
+
+    --size-subheading: 1.152rem;
+    --size-label: 1.152rem;
+}
+*,
+::before
+::after{
+    box-sizing: border-box;
+}
+
+body {
+    font-family: var(--font-family);
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 2;
+    color: var(--color-white);
+    background: linear-gradient(to right, var(--accent-left), var(--accent-right)), 
+    url("./https://www.oso-web.com/wp-content/uploads/2017/10/Web-Programmer-Tallents-1.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    color: var(--color-white);
+    margin: 0;
+}
+button,
+input,
+textarea,
+select {
+    font-family: inherit;
+}
+
+/* Utilites */
+
+.container{
+    margin: 5rem auto;
+}
+
+.text-center {
+    text-align: center !important;
+}
+
+.description {
+    font-size: var(--size-subheading);
+    margin: 1rem, auto;
+    text-shadow: 1px 1px #333;
+}
+
+header{
+    padding: 2rem 1rem 1rem 1rem;  
+}
+
+h1 {
+    font-weight: 400;
+    line-height: 1.2;
+    margin: 0;
+    text-shadow: 1px 1px #333333;
+}
+
+/* form */
+
+form {
+    background: var(--color-darkblue-alpha);
+    padding: 1rem 0.6rem;
+    border-radius: 0.3rem;
+    width: 50%;
+    margin: auto;
+}
+
+.form-group {
+    display: flex;
+    flex-direction: column;
+    margin: 1rem;
+}
+
+input[type="radio"],
+input[type="checkbox"]{
+    width: 1rem;
+    height: 1rem;
+}
+
+input[type="checkbox"]{
+    margin-right: 0.5rem;
+}
+
+.form-group p {
+    line-height: 1;
+    margin: 1rem 0.1rem;
+}
+
+.form-label {
+    font-size: var(--size-label);
+}
+
+.form-control {
+    display: block;
+    width: 100%;
+    font-size: 1rem;
+    height: 1rem;
+    padding: 0.375rem 0.75rem ;
+    border-radius: 0.3rem;
+    border: 1px solid #dddddd;
+    margin-top: 1rem;
+}
+
+button {
+background-color: var(--color-green);
+border: none;
+cursor: pointer;
+color: var(--color-white);
+padding: 0.7rem;
+
```

Comment:
please add a button hover effect you work good

Reply:
Nice observation! Kindly consider again explaining why the student should change his code, in this case, you could say to improve the project's alignment with the original design & improve the UX & UI. By doing this, you will make more explicit why a change is being requested and the student will have a better time correcting & understanding the piece of code 😄. Cheers and keep coding!